sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -327,38 +252,52 @@ void native_matmul(bfloat16 *A, bfloat16 *B, float *C) {
   }
 }
 
-int main(void) {
-  assert(MATRIX_SIZE >= tM && MATRIX_SIZE >= tK && MATRIX_SIZE >= tN &&
+template <typename T, typename TResult, size_t VNNI, size_t TM, size_t TN,
+          size_t TK, size_t MCache1, size_t NCache1, size_t KCache1,
+          size_t MCache2, size_t NCache2, size_t KCache2>
+void test() {
+  assert(MATRIX_SIZE >= TM && MATRIX_SIZE >= TK && MATRIX_SIZE >= TN &&
          "invalid matrix size");
-  assert((MATRIX_SIZE % tM) == 0 && (MATRIX_SIZE % tN) == 0 &&
-         (MATRIX_SIZE % tK) == 0 &&
-         "invalid matrix size detected: not a multiple of <tM,tN,tK>");
+  assert((MATRIX_SIZE % TM) == 0 && (MATRIX_SIZE % TN) == 0 &&
+         (MATRIX_SIZE % TK) == 0 &&
+         "invalid matrix size detected: not a multiple of <TM,TN,TK>");
+
+  std::cout << "Testing: " << TM << " x " << TN << " x " << TK
+            << " [TM x TN x TK]" << std::endl;
+  std::cout << "matrix size: " << MATRIX_SIZE << std::endl;
+  std::cout << "m-chache 1: " << MCache1 << std::endl;
+  std::cout << "n-chache 1: " << NCache1 << std::endl;
+  std::cout << "k-chache 1: " << KCache1 << std::endl;
+  std::cout << "m-chache 2: " << MCache2 << std::endl;
+  std::cout << "n-chache 2: " << NCache2 << std::endl;
+  std::cout << "k-chache 2: " << KCache2 << std::endl;
```

Comment:
I suggest removing it to reduce output noise. These parameters are the same for all MNK-combinations on one platform, so they do not provide more information about which particular kernel failed, outputting MNK is enough. 
```suggestion
```